src/k8s-runtime/azext_k8s_runtime/tests/latest/test_k8s_runtime.py

```diff
@@ -82,6 +83,32 @@ def test_storage_class_enable_disable_only_connected_cluster(self):
         self.cmd("az k8s-runtime storage-class enable --resource-uri {resource_uri}", expect_failure=True)
         self.cmd("az k8s-runtime storage-class disable --resource-uri {resource_uri}", expect_failure=True)
 
+    @live_only()
```

Comment:
since the test is marked as `@live_only()`, the recording file is useless and it is better to remove it.

Reply:
There are already recording files for live tests. Should I remove them as well?

Reply:
All recoding files are removed.